lib/polyfill/patchedmediakeys_webkit.js

```diff
@@ -31,22 +31,29 @@ goog.require('shaka.util.Uint8ArrayUtils');
  * Install a polyfill to implement {@link http://goo.gl/blgtZZ EME draft
  * 12 March 2015} on top of webkit-prefixed
  * {@link http://goo.gl/FSpoAo EME v0.1b}.
+ *
+ * @param {string} prefix
  */
-shaka.polyfill.PatchedMediaKeysWebkit.install = function() {
+shaka.polyfill.PatchedMediaKeysWebkit.install = function(prefix) {
   shaka.log.debug('PatchedMediaKeysWebkit.install');
 
-  goog.asserts.assert(HTMLMediaElement.prototype.webkitGenerateKeyRequest,
-                      'PatchedMediaKeysWebkit APIs not available!');
-
   // Alias.
   var PatchedMediaKeysWebkit = shaka.polyfill.PatchedMediaKeysWebkit;
+  PatchedMediaKeysWebkit.Prefix = prefix;
```

Comment:
This should be declared somewhere.  It should also be private to the class, which means it must end in an underscore.  For example:

/** @private {string} */
shaka.polyfill.PatchedMediaKeysWebkit.prefix_ = '';